content/dist/styleguide.css

```diff
@@ -814,6 +761,78 @@ time.time-reading, .time.time-reading {
   font-size: 1.55rem;
 }
 
+.socials-form {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -ms-flex-wrap: wrap;
+      flex-wrap: wrap
+}
+
+.socials-form li {
+  display: -webkit-box;
+  display: -ms-flexbox;
```

Comment:
flexbox is quite spread, isn't it?

Reply:
open in https://github.com/neomad-team/styleguide/issues/36